docs/guides/qunasys-quri-chemistry.ipynb

```diff
@@ -67,74 +60,270 @@
+    "\n",
+    "Post selection removes the erroneous states completely, leaving only the basis states to construct the subspace Hamiltonian. This usually saves classical resources, but might miss many states. Configuration recovery attempts to flip individual bits in erroneous basis states with respect to some probability distribution. Using configuration recovery increases the number of basis states to build a larger subspace Hamiltonian, but it uses a lot more classical resources.\n",
+    "\n",
+    "The options are explained in the following table. \n",
+    "\n",
+    "| Name                     | Type                        | Description                                                                                                                             | Required | Default |\n",
+    "| :----------------------- | :-------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------- | :------- | :------ |\n",
+    "| particle_number_symmetry | bool                        | If True, remove bit strings that does not respect particle number conservation from sampling result.                                    | No       | TRUE    |\n",
+    "| spin_symmetry            | bool                        | If True, remove bit strings that does not respect $s_z$ conservation from sampling result.                                              | No       | TRUE    |\n",
+    "| configuration_recovery   | Union[bool, dict[str, int]] | Activate configuration recovery if set to values other than False. Refer to the configuration recovery section for dict[str, int] input | No       | FALSE   |\n",
+    "\n",
+    "<Admonition type=\"caution\">You cannot set spin symmetry to True when particle symmetry is False. </Admonition>\n",
+    "\n",
+    "\n",
+    "#### Symmetry post selection\n",
+    "\n",
+    "These keys control the symmetry post selection: `particle_number_symmetry` and `spin_symmetry`. Setting `particle_number_symmetry` to True selects those states that conserve the total electron numbers.  Setting `spin_symmetry` to True selects those states that conserve the $s_z$ quantum number. Currently only $s_z = 0$ is supported. The default is:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mitigation_setting = {\n",
+    "    \"particle_number_symmetry\": True,\n",
+    "    \"spin_symmetry\": True,\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Configuration recovery\n",
+    "\n",
+    "Instead of the symmetry post selection, you can also choose to do configuration recovery. It can simply be turned on by setting it to `True` or customize it with a dictionary containing the following parameters:\n",
```

Comment:
```suggestion
    "Instead of the symmetry post selection, you can also choose to do configuration recovery. To turn it on, set it to `True`, and optionally customize it with a dictionary containing the following parameters:\n",
```